src/main/java/de/tum/in/www1/artemis/service/connectors/ProgrammingLanguageConfiguration.java

```diff
@@ -24,17 +24,20 @@ public class ProgrammingLanguageConfiguration {
      * @param buildImages the map of languages to build images
      */
     public void setImages(Map<String, String> buildImages) {
```

Comment:
I'm not sure if we also have to edit the Getter method?
Also I am still quite rusty with Java coming from Python so this is probably not the best solution?

Reply:
The name of the property is still "images". If so, no change should be needed.

Reply:
Not sure where the getter is being used. I had to change the setter because it couldn't handle the capitalization when loading the variables in spring boot apparently.

Reply:
The only important thing is that https://github.com/ls1intum/Artemis/pull/5295/files/084443f34a6b428a7774c10b79f05e20b7524123#diff-14960c11da0e755b78f23180b61f8b497d4a47987b4c29a7551d9ba6a4d0544eR64 works. it gets the docker image for a programming task by its type

Reply:
@dfuchss Yeah thx this is the Getter I thought of. Do you know how I can test this quickly? Probably a noob question but I'm not yet so experienced with Artemis and its features from a user perspective. I got a debugging setup but don't know how to trigger the getter :confused:

Reply:
The getter is used as soon as you create a programming task. It is used to determine the docker image used for building

Reply:
This is resovled as we are waiting for [PR 5596](https://github.com/ls1intum/Artemis/pull/5596) to be merged. Already also used the new env configs from that PR.